raiden-dapp/package.json

```diff
@@ -26,27 +26,30 @@
     "vue": "^2.6.10",
     "vue-class-component": "^7.1.0",
     "vue-i18n": "^8.15.0",
-    "vue-property-decorator": "^8.2.2",
+    "vue-property-decorator": "^8.3.0",
     "vue-router": "^3.1.3",
     "vue-virtual-scroller": "^1.0.0-rc.2",
     "vuetify": "2.1.9",
     "vuex": "^3.1.1"
   },
   "devDependencies": {
+    "@typescript-eslint/eslint-plugin": "^2.6.0",
+    "@typescript-eslint/parser": "^2.6.0",
```

Comment:
`@vue/eslint-config-typescript` depends on a really old version of this. This causes the warnings we get about unsupported versions. I did a quick check and everything looks ok. I am currently adding the dependency here, but we should keep an eye on the `@vue/eslint-config-typescript` and maybe remove the dependency.